### DISCUSSION TITLE
Error in Networks

### DISCUSSION THREAD

**Discussion by Losiek68** [rern/rAudio#556] [Fixed Issues]
Hi, 
I have found an error in Networks. My SSID password contains dollar sign "$" and when I choose Networks I got:
![image](https://user-images.githubusercontent.com/76816529/136930310-09472591-3871-482a-8ed7-8c4feb84808b.png)

I don't know when it broke but it used to work some time ago.

Regards

**Comment by Losiek68** [comment]
Just applied 20211020 and the problem still exist. Dollar sign in password phrase causes JSON error.

**Comment by R101G** [comment]
@Losiek68 
Have you tried double quotes around the password in the configuration file? I believe this is a cure for some similar problems.

**Comment by Losiek68** [comment]
I've got it double quoted:
![image](https://user-images.githubusercontent.com/76816529/138077641-abb386d4-6c2c-4bba-8037-b2fdac112480.png)

Of course Network works when I delete all "$" from password.

**Reply by R101G** [reply]
Sorry @Losiek68 , it's irrelevant now, but I meant put a second pair of quotation marks around the existing ones.

**Comment by rern** [comment]
Try:
Addons > rAudio 1 > long-press `Update` > ok for `UPDATE` branch

**Comment by Losiek68** [comment]
Works like a charm! 
Thank you.
